content/en/docs/collector/custom-collector.md

```diff
@@ -0,0 +1,161 @@
+
+Go ahead and create a manifest file named `builder-config.yaml` with the
+following content:
+
+> builder-config.yaml
+
+```yaml
+dist:
+  name: otelcol-dev
+  description: "Basic OTel Collector distribution for Developers"
+  output_path: ./otelcol-dev
+```
+
+Now you need to add the modules representing the components you want to be
+incorporated in this custom Collector distribution. Take a look at the
+[ocb configuration documentation](https://github.com/open-telemetry/opentelemetry-collector/tree/main/cmd/builder#configuration)
+to understand the different modules and how to add the components.
+
+We will be adding the following components to our development and testing
+collector distribution:
+
+- Exporters: Jaeger and Logging
+- Receivers: OTLP
+- Processors: Batch
+
+Here is what my `builder-config.yaml` manifest file looks after adding the
+modules for the components above:
+
+```yaml
+dist:
+  name: otelcol-dev
+  description: "Basic OTel Collector distribution for Developers"
+  output_path: ./otelcol-dev
+
+exporters:
+  - gomod:
+      "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/jaegerexporter
+      v0.53.0"
+  - import: go.opentelemetry.io/collector/exporter/loggingexporter
+    gomod: go.opentelemetry.io/collector v0.53.0
+
+receivers:
+  - import: go.opentelemetry.io/collector/receiver/otlpreceiver
+    gomod: go.opentelemetry.io/collector v0.53.0
+
+processors:
+  - import: go.opentelemetry.io/collector/processor/batchprocessor
+    gomod: go.opentelemetry.io/collector v0.53.0
+```
+
+## Step 3 - Generating the Code and Building your Collector's distribution.
+
+All you need now is to let the `ocb` do it's job, so go to your terminal and
+type the following command:
+
+```cmd
+ocb --config builder-config.yaml
+```
+
+If everything went well, here is what the output of the command should look
+like:
+
+```nocode
+2022-06-13T14:25:03.037-0500	INFO	internal/command.go:85	OpenTelemetry Collector distribution builder	{"version": "0.53.0", "date": "2022-06-08T15:05:37Z"}
+2022-06-13T14:25:03.039-0500	INFO	internal/command.go:108	Using config file	{"path": "builder-config.yaml"}
+2022-06-13T14:25:03.040-0500	INFO	builder/config.go:99	Using go	{"go-executable": "/usr/local/go/bin/go"}
+2022-06-13T14:25:03.041-0500	INFO	builder/main.go:76	Sources created	{"path": "./otelcol-dev"}
+2022-06-13T14:25:03.445-0500	INFO	builder/main.go:108	Getting go modules
+2022-06-13T14:25:04.675-0500	INFO	builder/main.go:87	Compiling
+2022-06-13T14:25:17.259-0500	INFO	builder/main.go:94	Compiled	{"binary": "./otelcol-dev/otelcol-dev"}
+```
+
+As defined in the `dist` section of your config file, you now have a folder
+named `otelcol-dev` containing all the source code and the binary for your
+Collector's distribution.
+
+You can now use the generated code to bootstrap your component development
+projects and easily build and distribute your own collector distribution with
+your components.
+
```

Comment:
Any "Next Steps", link to building your other how to on building receivers?

Reply:
We can definitely do that.
The Trace Receiver tutorial actually refers to this one, so we also need to make an update there once this one is merged.

Reply:
- Followup issue: #1502